modules/mono/glue/GodotSharp/GodotSharp/Core/GD.cs

```diff
@@ -455,6 +455,15 @@ public static uint Randi()
             return NativeFuncs.godotsharp_randi();
         }
 
+        /// <summary>
+        /// Returns a random boolean with a bias of <paramref name="chance"/> to be <see langword="true"/> and <see langword="false"/> otherwise.
+        /// </summary>
+        /// <returns>A random <see langword="bool"/>.</returns>
+        public static bool Randb(double p_chance = 0.5)
```

Comment:
We don't use `p_` prefixes in the C# API.

```suggestion
        public static bool Randb(double chance = 0.5)
```

Reply:
:0 Am I supposed to just know this stuff out of the box? Did I miss something in the contributing docs? 0.o (just pushed the new commit btw)

Reply:
It's just something to learn 🙂, but the style of the rest of the code is usually a good reference